src/containers/BooksForm.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { createBook } from '../actions';
+import { bookCategories, generateID } from '../utils';
+import '../styles/BooksForm.css';
+
+
+class BooksForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      title: '',
+      category: 'Action',
+      error: null,
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+
+  handleChange(event) {
+    const { name } = event.target;
+
+    this.setState({
+      [name]: event.target.value,
+    });
+  }
+
+  handleSubmit(event) {
+    // eslint-disable-next-line
+    const { title, category, error } = this.state;
+
+    // Validating the input form
+    if (title === '') {
+      this.setState({
+        error: 'Error: Title cannot be blank',
```

Comment:
## Necessary change
We should make sure the error state returns to null if the book is successfully submitted.